src/CachePlugin.php

```diff
@@ -379,7 +379,12 @@ private function createResponseFromCacheItem(CacheItemInterface $cacheItem)
 
         /** @var ResponseInterface $response */
         $response = $data['response'];
-        $response = $response->withBody($this->streamFactory->createStream($data['body']));
+        $stream = $this->streamFactory->createStream($data['body']);
+        if ($stream->isSeekable()) {
```

Comment:
I can, what kind of exception would you like to throw in this situation? I'm thinking of an `InvalidArgumentException`, as it would be a configuration issue IMO.